packages/TermsAndConditions/Footnote/Footnote.jsx

```diff
@@ -240,7 +241,7 @@ const Footnote = props => {
   // focus on the close button on open/content change
   useEffect(() => {
     if (content !== null && isVisible && closeRef && closeRef.current !== null) {
```

Comment:
Should the condition also check for `headingRef` instead of `closeRef` since you changed it?

```
(content !== null && isVisible && headingRef && headingRef.current !== null)
```

Reply:
Yes, in fact we don't need `closeRef` at all anymore. I've removed it completely.